tests/nnstreamer_filter_mxnet/Predictor.hh

```diff
@@ -0,0 +1,400 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
```

Comment:
1. Do you really need to add a Apache-2.0 file here?
2. Copying a whole file from other package is really not recommended. What's the package you are copying from?

Reply:
As I mentioned in the PR message and in the commit message, these two problematic files are directly copied & modified from [the MXNet reference (example) implementation](https://github.com/apache/incubator-mxnet/blob/1.7.0/cpp-package/example/inference) and I already highly doubt these are proper files to be merged.

I used them to compare the inference results between the official example code and my nnstreamer port. Since it at least triggers the license issue, I will remove them completely when I start writing proper unit tests.